packages/core-svelte/src/lib/components/LimitedRadioComponent.svelte

```diff
@@ -98,32 +99,41 @@
             }
         }
     };
+
+    const onClickLabel = (event: MouseEvent & { currentTarget: EventTarget & HTMLLabelElement }) => {
+        event.stopPropagation();
+    };
+
 </script>
 
 <span
-    role="radiogroup"
-    aria-labelledby={ariaLabel}
-    {id}
-    class="limited-radio-component-group {box.cssClass}"
-    class:limited-radio-component-vertical={!isHorizontal}
+  role="radiogroup"
+  aria-labelledby={ariaLabel}
+  {id}
+  class="freon-radio-group limited-radio-component-group  {box.cssClass}"
+  class:freon-radio-group-vertical={!isHorizontal}
 >
     {#each myEnum as nn, i}
-        <span class="limited-radio-component-single">
-            <md-radio
-                id="{id}-{nn}-{i}"
-                name="{id}-group"
-                role="radio"
-                tabindex="0"
-                aria-checked={currentValue === nn}
-                value={nn}
-                checked={currentValue === nn}
-                aria-label="radio-control-{nn}"
-                onclick={onClick}
-                onchange={onChange}
-                onkeydown={onKeyDown}
-                bind:this={allElements[i]}
-            ></md-radio>
-            <label class="limited-radio-component-label" for="{id}-{nn}-{i}">{nn}</label>
+        <span class="freon-radio-item limited-radio-component-single">
+            <!-- svelte-ignore a11y_no_noninteractive_element_interactions -->
+            <!-- svelte-ignore a11y_click_events_have_key_events -->
+            <label class="freon-radio-label limited-radio-component-label"   onclick={onClickLabel}>
+                <input
+                  class="limited-radio-component-input"
```

Comment:
This class is missing in the boolean radio component. Should it be there as well for balance?

Reply:
In the custom components (samples/CourseSchedule) there are still smui dependencies.
Not a problem right now, but we might want to change these in the future.